libcontainer/nsenter/clone.c

```diff
@@ -0,0 +1,39 @@
+#define _GNU_SOURCE
+#include <sched.h>
+#include <errno.h>
+#include "clone.h"
+
+/* Assume the stack grows down, so arguments should be above it. */
```

Comment:
This assumption seems untrue for HP PA (PA-RISC) https://man7.org/linux/man-pages/man2/clone.2.html

I don't think we will ever support PA-RISC, but probably it should be mentioned here